projects/packages/publicize/src/class-publicize-base.php

```diff
@@ -244,10 +244,6 @@ public function __construct() {
 		// Alter the "Post Publish" admin notice to mention the Connections we Publicized to.
 		add_filter( 'post_updated_messages', array( $this, 'update_published_message' ), 20, 1 );
 
-		// Connection test callback.
-		// TODO Get rid of this and the method after April 2025.
```

Comment:
Ok, I think this might answer some of my questions about conflicts

Reply:
Yes, I gave it a second thought an then realized that publicize package is always going to be the latest version.